scripts/change-extensibility.js

```diff
@@ -0,0 +1,72 @@
+if(!process.env.npm_config_plugin) {
+    throw new Error("Missing plugin name.");
+}
+
+let pluginSpaceName = process.env.npm_config_plugin;
+let metadata = process.env.npm_config_name;
+let pluginForgeVersion = process.env.npm_config_forge
+let mabsMin = process.env.npm_config_mabs;
+
+let repository = process.env.npm_config_repository;
+let branch = process.env.npm_config_branch;
+
+let environment = process.env.npm_config_environment;
+
+if(process.env.npm_config_authentication == null) {
+    throw new Error("Missing authentication argument.");
+}
+let basicAuthentication = process.env.npm_config_authentication;
+
+let url = `https://${environment}/CodeUpdater/rest/Bulk/ExtensabilityUpdate`;
+
+let extensibilityChangeJson = {
```

Comment:
We should also consider the `AppShieldObfuscationRules` android preference that some plugins can have right?

Reply:
I think we don't want to have that logic here. That's more of a change in the `CodeUpdater` code >> it should update the Extensibility JSON and not override it completely.

Reply:
Ok ok, that makes sense.